lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -676,6 +647,22 @@ export abstract class ClientApplication {
         }
     }
 
+    /**
+     * Returns the signed in account matching the given session id,
+     * or null when no matching account is found.
+     * Note: sid must be enabled as an optional id token claim.
+     * @param sid Session ID (optional id token claim)
+     * @returns {@link AccountInfo} - the account object stored in MSAL
+     */
+    public getAccountBySessionId(sid: string): AccountInfo | null {
+        const allAccounts = this.getAllAccounts();
+        if (sid && allAccounts && allAccounts.length) {
```

Comment:
`getAllAccounts` should always return type array, length check should be enough
```suggestion
        if (sid && allAccounts.length) {
```

Reply:
Indeed. This was copy pasta from one of the other `getAccountBy` methods, so I'll update those accordingly as well.